DEVELOPMENT_CONTRIBUTION.md

```diff
@@ -122,7 +122,7 @@ Set scope of the commit if possible:
   datagrid  date-picker  dropdown   form          grid          header        i18n   input
   label     list         login      modal         password      progress-bar  radio  select
   sidenav   signpost     spinner    stack-view    stepper       table         tabs   textarea
-  timeline  toggle       tooltip    tree-view     vertical-nav  wizard        icons
+  timeline  toggle       tooltip    tree-view     vertical-nav  wizard        icons  website
```

Comment:
We don't want website as a scope, its not a component. You would just use `docs: whatever I did` instead.

Reply:
Also `build`